components/scripts/lib/parse.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
-1: I recommend naming this `build-scan-parse.sh` so that it's clear this is still focused on build scans.

Reply:
Addressed by fa178a1c3b65193c584d280eea252be392953871